e2e_tests/docker-compose/compose.yml

```diff
@@ -0,0 +1,135 @@
+version: "3"
+services:
+  chain:
+    container_name: chain
+    image: ghcr.io/foundry-rs/foundry
+    hostname: chain
+    platform: linux/amd64
+    ports:
+      - "8545:8545"
```

Comment:
Would be nice if we could run all of these tests and envs in parallel. One thing that we'd need for that is exposing different (random) ports per test